### ISSUE TITLE
Add infinitif to anki

### ISSUE THREAD

**Issue by ShadowMarter** [simjanos-dev/LinguaCafe#365]
Currently, the popup dictionary shows the infinitif version of a word, but you can only export the conjugated version to anki. Is it possible to make the dictionary manually editable before anki sync?

**Comment by simjanos-dev** [comment]
Yeah, that's a great idea. I'll add an option for a popup edit before sending it to Anki.
